blobs.go

```diff
@@ -65,10 +63,10 @@ func (r *blobsTable) TransformExpressionsUp(f func(sql.Expression) (sql.Expressi
 	return r, nil
 }
 
-func (r blobsTable) RowIter(_ sql.Session) (sql.RowIter, error) {
+func (r blobsTable) RowIter(session sql.Session) (sql.RowIter, error) {
 	iter := new(blobIter)
 
-	repoIter, err := NewRowRepoIter(r.pool, iter)
+	repoIter, err := NewRowRepoIter(session, iter)
```

Comment:
since the only thing the row repo iter needs from the session is the pool, what do you think about keeping it how it was and just extracting the pool from the session in RowIter? It's a bit of repeated code over all RowIter's, but I feel a bit odd tht RowRepoIter needs the session

Reply:
It also needs the context to cancel the iterator:

https://github.com/jfontan/gitquery/blob/626a886bcfb598eaca5195a02074a2fc9a550d12/repository_pool.go#L233

Reply:
Ah, ok. Then ignore my comment 😄